vec/vec-v12x/src/main/java/com/foursoft/harness/vec/v12x/navigations/CustomPropertyNavs.java

```diff
@@ -107,4 +108,13 @@ public static Function<VecLocalizedStringProperty, Optional<String>> convertLoca
                 .map(VecLocalizedString::getValue);
     }
 
+    public static <T extends VecCustomProperty> Function<Collection<VecCustomProperty>, Optional<T>> customPropertyOfType(
```

Comment:
This should either return a List<T> and not use findFirst or make it clear in the name that it expects 0 or 1 elements and if more are found, throws an exception.